src/mcpp.cpp

```diff
@@ -45,14 +45,15 @@ Coordinate MinecraftConnection::getPlayerPosition() {
 }
 
 void MinecraftConnection::setPlayerTilePosition(const Coordinate& tile) {
-    conn->sendCommand("player.setTile", tile.x, tile.y, tile.z);
+    Coordinate newTile = tile;
+    newTile.y++;
+    conn->sendCommand("player.setTile", newTile.x, newTile.y, newTile.z);
```

Comment:
Why is `getPlayerTilePosition` done in terms of `getPlayerPosition` but this is done manually? Ideally this would be `setPlayerPosition(tile + Coord{ 0, 1, 0})` or something similar.

Reply:
Thanks for catching it, will resolve this right now.